packages/@lwc/synthetic-shadow/src/faux-shadow/shadow-token.ts

```diff
@@ -5,7 +5,6 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 import { isUndefined, defineProperty } from '@lwc/shared';
-import { setAttribute, removeAttribute } from '../env/element';
```

Comment:
Looks like this was the last remaining usage of removeAttribute. Please clean up `removeAttribute` in `../env/element`.